arches_component_lab/src/arches_component_lab/cards/DefaultCard/DefaultCard.vue

```diff
@@ -84,15 +84,17 @@ watchEffect(async () => {
                 props.graphSlug,
                 props.nodegroupAlias,
             );
-        const tileDataPromise = fetchTileData(
-            props.graphSlug,
-            props.nodegroupAlias,
-            props.tileId,
-        );
+        if (props.tileId && props.tileId !== "root") {
```

Comment:
I don't _love_ needing this guard in `defaultCard.vue`. Happy to replace it with a UUID check && throw an Error though, but it feels like this should be guarded at a different level

Reply:
Where is tileId even being set to the string "root"?  I tend to agree with @chrabyrd.  I would definitely not expect tileId to be anything other than None or a uuid.

Reply:
agree! https://github.com/archesproject/arches-modular-reports/pull/28